### PR TITLE
Mobile app v2 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/deploy/mobile-app-changelog.md [hunk 1]
```diff
@@ -2,12 +2,40 @@
...
+ - Improved network reliability reducing app hang and errors if internet connectivity is unstable.
+
+#### Improved Performance
+ - The app is more responsive to user input, launches faster and consumes less memory when running.
+
+#### Improved Stability
+ - Improved app stability so users encounter fewer crashes and bugs.
+
+### Known Issues
+ - On iOS, the icon badge number may not reflect the correct mention count accross all servers.
```
Comment (enahum) [inline source/deploy/mobile-app-changelog.md:36]: This should not really be the case anymore

File: source/deploy/mobile-app-changelog.md [hunk 2]
```diff
@@ -2,12 +2,40 @@
...
+ - Channel sidebar disappears sometimes when the channel categories are not fetched from the server.
```
Comment (enahum) [inline source/deploy/mobile-app-changelog.md:37]: This is definitely not the case anymore

File: source/deploy/mobile-app-changelog.md [hunk 3]
```diff
@@ -2,12 +2,40 @@
...
+ - Posts sometimes get stuck behind the post textbox on iPad.
```
Comment (enahum) [inline source/deploy/mobile-app-changelog.md:38]: Also not the case

**Review by enahum — DISMISSED** [review]
Not sure if @esethna or @Willyfrog wants to add something here

File: source/deploy/mobile-app-changelog.md [hunk 4]
```diff
@@ -2,12 +2,38 @@
...
+- Server Versions Supported: Server v7.1.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v7.1.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) 6.3.0 has ended and upgrading to server ESR v7.1.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Highlights
+
+#### Multiple Workspaces
```
Comment (esethna) [inline source/deploy/mobile-app-changelog.md:23]: @amyblais propose we replace the highlights section with the first few paragraphs of this blog: https://mattermost.com/blog/preparing-for-mobile-v2-0/

Lets also link to that blog for more details in the compatibility section

File: source/deploy/mobile-app-changelog.md [hunk 5]
```diff
@@ -2,12 +2,38 @@
...
+ - Improved network reliability reducing app hang and errors if internet connectivity is unstable.
+
+#### Improved Performance
+ - The app is more responsive to user input, launches faster and consumes less memory when running.
+
+#### Improved Stability
+ - Improved app stability so users encounter fewer crashes and bugs.
+
+### Known Issues
+ -
```
Comment (esethna) [inline source/deploy/mobile-app-changelog.md:36]: Let's fill this in from any high impact known bugs that we don't fix by end of next week

Comment (esethna) [reply source/deploy/mobile-app-changelog.md:36]: https://mattermost.atlassian.net/jira/software/c/projects/MM/boards/101

Comment (Willyfrog) [reply source/deploy/mobile-app-changelog.md:33]: should we mention that the ipad mini might behave as a phone?

Comment (enahum) [reply source/deploy/mobile-app-changelog.md:33]: No cause is not true

File: source/deploy/mobile-app-changelog.md [hunk 6]
```diff
@@ -2,12 +2,35 @@
...
+- [1.54.0 Release](#id8)
+- [1.53.0 Release](#id13)
+- [1.52.0 Release](#id17)
+
+## 2.0.0 Release
+- Release Date: December 15, 2022
+- Server Versions Supported: Server v7.1.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - Please [see here](https://mattermost.com/blog/preparing-for-mobile-v2-0/) for more details on the v2 release.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:17]:
```suggestion
 - Please [see here](https://mattermost.com/blog/preparing-for-mobile-v2-0/) for more details on preparing for the mobile v2 release.
```

File: source/deploy/mobile-app-changelog.md [hunk 7]
```diff
@@ -2,12 +2,35 @@
...
+ - **Upgrade to server version v7.1.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) 6.3.0 has ended and upgrading to server ESR v7.1.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:18]: The server version details are mentioned twice -- here and directly under the section heading. Is the repetition intentional?

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline

File: source/deploy/mobile-app-changelog.md (hunk 5 above)
Comment (enahum) [reply source/deploy/mobile-app-changelog.md:33]: https://community.mattermost.com/core/pl/c36kbaoq7idzxdu654u7dcee5c

**Review by Willyfrog — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md [hunk 8]
```diff
@@ -2,12 +2,36 @@
...
+
+Mattermost mobile v2.0 is a major update to the iOS and Android apps bringing support for multiple workspaces and a more stable and performant user experience that includes:
+
+ - Support for collaborating in multiple workspaces with a redesigned channel and team sidebar for easier navigation. 
+ - Improved app responsiveness to user input, faster launch, and less memory consumed when running.
+ - Improved app stability so users encounter fewer crashes and bugs, as well as reduced app hang and errors if internet connectivity becomes unstable. 
+
+Users now gain a more reliable and feature-rich application, improving their experience for collaborating with their teams on the go.
+
+### Known Issues
```
Comment (esethna) [inline source/deploy/mobile-app-changelog.md:32]: Need to fill this out more in early Jan based on a review of the high impact bugs in the queue and int he v2.1 fix version

File: source/deploy/mobile-app-changelog.md (hunk 7 above)
Comment (esethna) [reply source/deploy/mobile-app-changelog.md:18]: I think the repetition in this case is okay to drive the point home ;)

File: source/deploy/mobile-app-changelog.md [hunk 9]
```diff
@@ -2,12 +2,36 @@
...
-- [1.51.2 Release](#id18)
+- [2.0.0 Release](#release)
+- [1.55.1 Release](#id1)
+- [1.55.0 Release](#id4)
+- [1.54.0 Release](#id8)
+- [1.53.0 Release](#id13)
+- [1.52.0 Release](#id17)
+
+## 2.0.0 Release
+- Release Date: December 15, 2022
```
Comment (esethna) [inline source/deploy/mobile-app-changelog.md:13]: Jan

**Review by esethna — CHANGES_REQUESTED** [review]

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**